ckanext/canada/tests/test_ati.py

```diff
@@ -1,55 +1,97 @@
 # -*- coding: UTF-8 -*-
-from nose.tools import assert_equal, assert_raises
 from ckanapi import LocalCKAN, ValidationError
 
-from ckan.tests.helpers import FunctionalTestBase
+import pytest
+from ckan.tests.helpers import reset_db
 from ckanext.canada.tests.factories import CanadaOrganization as Organization
 
 from ckanext.recombinant.tables import get_chromo
 
-class TestAti(FunctionalTestBase):
-    def setup(self):
-        super(TestAti, self).setup()
+import logging
+log = logging.getLogger(__name__)
+
+
+class TestAti(object):
+    @classmethod
+    def setup_class(self):
+        """Method is called at class level before all test methods of the class are called.
+        Setup any state specific to the execution of the given class (which usually contains tests).
+        """
+        reset_db()
+
+        log.info('Running setup for {}'.format(self.__name__))
+
         org = Organization()
         lc = LocalCKAN()
+
+        log.info('Creating organization with id: {}'.format(org['name']))
+        log.info('Setting organization dataset type to {}'.format('ati'))
```

Comment:
```suggestion

        org = Organization()
        lc = LocalCKAN()

```

Reply:
debug logging probably not required here

Reply:
Logging removed.